src/WaCore.Sample/Startup.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using WaCore.Sample.Data;
+using Microsoft.EntityFrameworkCore;
+using WaCore.Sample.Data.Repositories;
+
+namespace WaCore.Sample
+{
+    public class Startup
+    {
+        public Startup(IConfiguration configuration)
+        {
+            Configuration = configuration;
+        }
+
+        public IConfiguration Configuration { get; }
+
+        // This method gets called by the runtime. Use this method to add services to the container.
+        public void ConfigureServices(IServiceCollection services)
+        {
+            services.AddMvc();
+
+            var connection = @"Server=(localdb)\mssqllocaldb;Database=WaCore.Sample;Trusted_Connection=True;";
```

Comment:
I just tried. It's really easy - you just change it from ``UseSqlServer(connectionString)`` to ``UseInMemoryDatabase("databaseName")``
But I don't like having it in the Sample app. I want to use this app also for manual testing as I code and then I don't want the database to be wiped every time I change something in code and refresh the page. With InMemory it wipes it because it restarts the application.